qCC/ccGraphicalSegmentationTool.cpp

```diff
@@ -60,6 +60,9 @@
 #include <omp.h>
 #endif
 
+std::set<ccHObject*> enableAtTheClose;
```

Comment:
Are those static?! Consider using members of the tool instance instead.

And rename them m_enableOnClose and m_disableOnClose